taxi/forms.py

```diff
@@ -0,0 +1,25 @@
+from django.contrib.auth.forms import UserCreationForm
+from django.forms import ModelForm, CheckboxSelectMultiple
+
+from taxi.models import Driver, Car
+
+
+class DriverCreationForm(UserCreationForm):
+    class Meta(UserCreationForm.Meta):
+        model = Driver
+        fields = UserCreationForm.Meta.fields + (
+            "first_name", "last_name", "license_number"
+        )

```

Comment:
The `DriverCreationForm` should include validation for the `license_number` field to ensure it consists of 3 uppercase letters followed by 5 digits. This is a critical requirement as per the task description.